.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+  - repo: https://github.com/ambv/black
```

Comment:
The repo has moved to https://github.com/psf/black

Reply:
Updated!